src/scene_battle_rpg2k3.cpp

```diff
@@ -740,8 +740,6 @@ bool Scene_Battle_Rpg2k3::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBas
 							action->IsPositive() ? Font::ColorHeal : Font::ColorDefault,
 							std::to_string(action->GetAffectedHp()));
 					}
-
-					target->BattlePhysicalStateHeal(action->GetPhysicalDamageRate());
```

Comment:
> Refactor BattlePhysicalStateHeal 

"Remove call from battle 2k3 that did nothing"

Did this break now autostatehealing in 2k3 or where is this handled?

Reply:
This method returns a vector of conditions to heal from a physical attack. I have no idea why it's here in 2k3 battle, as it just computes this vector and does nothing with it.

Physical states healing is all setup properly in `Game_BattleAlgorithm`.

Auto state healing is a different concern. I haven't checked where that is in 2k3, but this change doesn't affect that either way.